### PR TITLE
Fixes #1238 remove column

### LINKED ISSUE #1238: Remove column from data repository

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/OSPSuite.Assets/UIConstants.cs [hunk 1]
```diff
@@ -1295,16 +1296,16 @@ public static string UnsupportedFileFormat(string fileName)
       public static readonly string InconsistentMoleculeAndMolWeightException = "Molecule and Molecular Weight do not match. Please either edit your Molecule or your Molecular Weight in your project or remove the Molecular Weight from your mappings";
       public static readonly string InvalidMixOfSimulationAndSimulationBatch = "You already have Simulation and SimulationBatch objects and should not mix, please invoke Clear to start adding objects from a fresh start";
       public static readonly string MismatchingArrayLengths = "Arrays should have the same length";
-      public static string EmptyDataSet(string dataSetName) => $"Your settings ended up with folowwing empty datasets: '{dataSetName}'. Please remove the data set from your data, filter it out or add at least one observation for it.";
+      public static string EmptyDataSet(string dataSetName) => $"Your settings ended up with following empty datasets: '{dataSetName}'. Please remove the data set from your data, filter it out or add at least one observation for it.";
```
Comment (msevestre) [inline src/OSPSuite.Assets/UIConstants.cs:1299]: some typos here

File: src/OSPSuite.R/Services/DataRepositoryTask.cs [hunk 2]
```diff
@@ -39,6 +41,15 @@ public interface IDataRepositoryTask
...
       DataColumn AddErrorColumn(DataColumn column, string name, string errorType);
 
+      /// <summary>
+      ///    Removes the <paramref name="column" /> from the <paramref name="dataRepository" />. if the column is a related
+      ///    column, the association will also be removed. If the column is a base grid column, it can only be removed if the
+      ///    data repository has no other column
+      /// </summary>
+      /// <param name="dataRepository">DataRepository containing the column to remove</param>
+      /// <param name="column">Column to remove</param>
+      void RemoveColumn(DataRepository dataRepository, DataColumn column);
```
Comment (msevestre) [inline src/OSPSuite.R/Services/DataRepositoryTask.cs:51]: @PavelBal This is the method to use to remove a column from a data repository

**Review by msevestre — COMMENTED** [review]
No need for a workaround.

**Review by Yuri05 — APPROVED** [review]

File: src/OSPSuite.R/Services/DataRepositoryTask.cs (hunk 2 above)
Comment (PavelBal) [reply src/OSPSuite.R/Services/DataRepositoryTask.cs:51]: Love the comments!

**Pull request merged**